gengo/gengo.py

```diff
@@ -212,6 +212,8 @@ def get(self, **kwargs):
                 post_data['action'] = kwargs.pop('action')
             if 'job_ids' in kwargs:
                 post_data['job_ids'] = kwargs.pop('job_ids')
+            if 'attachment' in kwargs:
```

Comment:
I'm not sure myself but I'm guessing [from the code in G here](https://github.com/gengo/g/blob/739e08225d8f1b9e3b591285d9ab9128d3d9c540/api/controllers/comment.go#L46), would the *keys* in the payload be this instead, perhaps? :

```json
{
    "body": "I love lamp!", 
    "context": "something",
    "attachments": ["./testfiles/test_file1.txt", "./testfiles/test_file2.txt"]
}
```

as such, would it be :
```diff
- attachment
+ attachments
- comment
+ body
```